### ISSUE TITLE
Remove codes for direct execution of test methods ("if __name__ == '__main__':")

### ISSUE THREAD

**Issue by wmnsk** [kbandla/dpkt#319]
Since the introduction of the `absolute_import`, the following lines of code, which exist in almost all the modules, no longer have made any sense, hasn't it? Rather, I think it should be removed, as it decreases test coverages.

```python
if __name__ == '__main__':
    test_pack()
    test_unpack()
    print('Tests Successful...')
```

Are there any concern for removing them to improve the score of coveralls?

**Comment by kbandla** [comment]
Good point @wmnsk. Lets see some stats. 
I just went though the coverage [numbers](https://coveralls.io/github/kbandla/dpkt) 

* 16 modules- 100%
* 14 modules - [90-99]%
* 18 modules - [80-89]%
* 05 modules - [70-79]%
* 08 modules - [60-69]%
* 08 modules - [50-59]%
* 05 modules - [40-49]%
* 02 modules - [30-39]%

We can surely pull up the coverage numbers to 100% for one module. But the others will still stay in the same range mostly. I think we have to improve tests for the modules a lot. Then we can surely take off the if/main to reach the 100% goal :)

For those who want to keep the if/main, the only reason I can think is that it helps running the tests (a) without pytest installed, and (b) makes unittesting individual modules during dev easy. 

Curious what the others think

**Comment by wmnsk** [comment]
@kbandla Thank you for the stats.
I think excluding these lines from coverage checking scope is alternative way to fix this issue(I don't know if we can add such exceptions into coveralls, though).

Let's listen to the others' opinion, anyway.

**Comment by obormot** [comment]
Every pull request now says the coverage has decreased -0.01% because the ```if __name__ == '__main__'``` block is no longer executed. This is pretty annoying if the change is small and doesn't add tests increasing the coverage.

**Comment by crocogorical** [comment]
Resolved in #528
